CDC.WebNotes.DB.Migrations/Migrations/20220717_InitialSchema.cs

```diff
@@ -11,7 +11,7 @@ public class InitialSchema : Migration
         public override void Up()
         {
             Create.Table(NotesTableName)
-                .WithColumn(IdColumnName).AsInt32().PrimaryKey().Identity()
+                .WithColumn(IdColumnName).AsInt32().NotNullable().PrimaryKey().Identity()
```

Comment:
1. It does not work sutch way. We always cteate new migrations to update DB schema 
2. PRimary key should be not nullvle anyway